IPIP/0000-gateway-tar-response-format.md

```diff
@@ -53,11 +53,22 @@ This RFC is backwards compatible.
 Manually created UnixFS DAGs can be turned into malicious TAR files. For example,
 if a UnixFS directory contains a file that points at a relative path outside of
 its root, the unpacking of the TAR file may overwrite local files. In order to prevent
-this, all relative paths in the produced TAR **MUST** be sanitized/truncated to never
-go beyond the root of the exported directory.
+this, if the UnixFS directory contains a file that points at a relative path outside
+of the root, the TAR file creation **MUST** fail.
+
+To test this, we provide two car files:
+
+* ✅ [inside-root.car](0000-gateway-tar-response-format/inside-root.car) is a UnixFS DAG that contains
+a file with a relative path that points inside the root directory. Downloading it as a TAR must work.
+* ❌ [outside-root.car](0000-gateway-tar-response-format/outside-root.car) is a UnixFS DAG that contains
+a file with a relative path that points outside the root directory. Downloading it as a TAR must error.
```

Comment:
@hacdias nit: please do not include CARs in the repo, only CIDs. :pray: 
We want to follow that practice so the specs repo do not baloons over time in size.